mkdocs/docs/tutorials/gitops/argocd.md

```diff
@@ -0,0 +1,211 @@
+
+    ![CICD](../../images/CICD.png)
+
+!!! tip "Scenario: Iter8+Gitops"
+    Iter8 can be used in the context of GitOps (shown below) so that new versions of an application can be progressively rolled out, or even rolled back when problems are detected. In this tutorial, we will use Argo CD as the CD pipeline tool and Istio as the underlying service mesh,
+
+    ![CICD+Iter8](../../images/CICD+Iter8.png)
+
+    We assume the reader has at least a basic understanding of how Iter8 works from the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster). Since the Env repo is at the heart of GitOps, we will focus mainly on how to setup and manage the Env repo during application update. In this tutorial, we will cover the following topics.
+
+    1. How to setup an Env repo to work with Iter8+GitOps
+    2. How to update the Env repo to start an Iter8 experiment
+    3. How to cleanup the Env repo after an Iter8 experiment is finished
+
+## Step 1. Create K8s cluster
+
+If you don't already have a K8s cluster, create a Minikube or Kind K8s cluster locally by referring to step 1 of the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster). Otherwise, skip to the next step.
+
+## Step 2. Fork repo
+
+As you will need to make changes to the Env repo to test new app versions, you will need your own copy of the repo.
+
+* Fork this repo: [https://github.com/iter8-tools/iter8](https://github.com/iter8-tools/iter8)
+* Now you should have your own Env repo at: https://github.com/[YOUR_ORG]/iter8
+
+## Step 3. Install platform components
+
+In your K8s cluster, you will need to install Istio, Prometheus, Iter8, and Argo CD. Run the following script to install these:
+
+```shell
+git clone https://github.com/[YOUR_ORG]/iter8.git
+cd iter8
+export ITER8=$(pwd)
+$ITER8/samples/gitops/platformsetup.sh
+```
+
+replacing `[YOUR_ORG]` with your Github organization name, aka, your Github username. Now, do the same replacement operation to update some references in the repo so they will point at your forked repo.
```

Comment:
Is it always your github username? perhaps say "Github organization or username"

Reply:
The existing wording "... with your Github organization name, aka, your Github username" seems to mean the same thing to "Github organization or username", but slightly wordy. I can make the change.